src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py

```diff
@@ -155,41 +209,144 @@ def test_sp_show_exit_code(self):
 class ApplicationScenarioTest(ScenarioTest):
 
     def test_application_scenario(self):
-        display_name = self.create_random_name(prefix='azure-cli-test-graph-app', length=30)
+        """
+        - Test creating application with its properties.
+        - Test creating application first and update its properties.
+        """
+        display_name = self.create_random_name(prefix='azure-cli-test', length=30)
 
         # identifierUris must be on verified domain
         # https://docs.microsoft.com/en-us/azure/active-directory/develop/security-best-practices-for-app-registration#appid-uri-configuration
+        self.kwargs.update({
+            'display_name': display_name,
+            'identifier_uri': f'api://{display_name}',
+            'homepage': 'https://myapp.com/',
+            'web_redirect_uri_1': 'http://localhost/webtest1',
+            'web_redirect_uri_2': 'http://localhost/webtest2',
+            'public_client_redirect_uri_1': 'http://localhost/publicclienttest1',
+            'public_client_redirect_uri_2': 'http://localhost/publicclienttest2',
+            'app_roles': TEST_APP_ROLES,
+            'optional_claims': TEST_OPTIONAL_CLAIMS
+        })
 
-        # tenant_id will be 0000... in playback mode, so register it in name_replacer
-        tenant_id = self.cmd('account show --query tenantId').get_output_in_json()
-        self.name_replacer.register_name_pair(tenant_id, MOCKED_TENANT_ID)
+        # Create
+        result = self.cmd(
+            'ad app create --display-name {display_name} '
+            '--identifier-uris {identifier_uri} '
+            '--is-fallback-public-client True '
+            '--sign-in-audience AzureADMultipleOrgs '
+            # web
+            '--web-home-page-url {homepage} '
+            '--web-redirect-uris {web_redirect_uri_1} {web_redirect_uri_2} '
+            '--enable-access-token-issuance true --enable-id-token-issuance true '
+            # publicClient
+            '--public-client-redirect-uris {public_client_redirect_uri_1} {public_client_redirect_uri_2} '
+            "--app-roles '{app_roles}' "
+            "--optional-claims '{optional_claims}'",
```

Comment:
We need to make sure all arguments are tested since the new command argument coverage test will spot the missing arguments!